department-of-veterans-affairs/caseflow
# Queue | Use cached appeal for detail view

### Description
Currently, when the user navigates to `QueueDetailView` (appeal detail), we [always fetch the appeal from the server](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/queue/QueueLoadingScreen.jsx#L56). This checks if the appeal to display is already loaded, and uses the cached version if so.

### Acceptance Criteria 
- [ ] Navigating from `/queue` to any appeal detail view does not load that appeal from the server

### Testing Plan
1. Go to `/queue` as an attorney
2. Click on case detail link, confirm no request is made to `/appeals/${vacolsId}`

![use_cached_appeal](https://user-images.githubusercontent.com/8533004/40497145-77128b26-5f49-11e8-8082-a985cbd30ffb.gif)
